lib/WUI/http/httpd_structs.h

```diff
@@ -28,7 +28,9 @@ static const char *const g_psHTTPHeaderStrings[] = {
     "Connection: keep-alive\r\nContent-Length: ",
     "Server: " HTTPD_SERVER_AGENT "\r\n",
     "\r\n<html><body><h2>404: The requested file cannot be found.</h2></body></html>\r\n",
-    "HTTP/1.0 500 Internal Server Error\r\n"
+    "HTTP/1.0 500 Internal Server Error\r\n",
+    "HTTP/1.0 401 Unauthorized\r\nWWW-Authenticate: ApiKey realm=\"401\"",
```

Comment:
/question/ Shouldn't `WWW-Authenticate: ApiKey realm=` be handled separately from the standard 401 status code?

Reply:
The possibility of the deployed web server are significantly limited so there must be a choice how to balanced new functionality with complexity of rework/rewrite of the existing web server.